pkg/gcp/client.go

```diff
@@ -178,6 +182,34 @@ func (c *gcpClient) DeleteWorkloadIdentityPool(ctx context.Context, resource str
 	return c.oldIamClient.Projects.Locations.WorkloadIdentityPools.Delete(resource).Context(ctx).Do()
 }
 
+func (c *gcpClient) EnableServiceAccount(
+	ctx context.Context,
+	serviceAccountId string,
+	projectId string,
+) error {
+	_, err := c.oldIamClient.Projects.ServiceAccounts.Enable(
+		FmtSaResourceId(serviceAccountId, projectId),
+		&iamv1.EnableServiceAccountRequest{},
+	).Do()
+	if err != nil {
+		return c.handleEnableServiceAccountError(err)
+	}
+	return nil
+}
+
+func (c *gcpClient) EnableWorkloadIdentityPool(
+	ctx context.Context,
+	poolId string,
+) error {
+	_, err := c.oldIamClient.Projects.Locations.WorkloadIdentityPools.Patch(
+		poolId,
+		&iamv1.WorkloadIdentityPool{
+			Disabled: false,
+		},
+	).UpdateMask("disabled").Do()
+	return err
```

Comment:
does this err not have to be handled same as  other error msgs  line @ line#195

Reply:
ack.